### ISSUE TITLE
Feature Request: Support for Star Wars 5e 

### ISSUE THREAD

**Issue by gravagehulk** [Feu-Secret/Tokenmagic#136]
Is there any way to enable this for Star Wars 5e (SW5e)? The rule system is basically a copy/paste of the 5e rules. It has things like flame throwers that could benefit from the animated fire template. I've been testing the system for a few weeks and it's rock solid. The developer of Automated Animations was able to add support within a few minutes, due to the similarity to 5e ruleset.

https://github.com/unrealkakeman89/sw5e

Thank you!

**Comment by Feu-Secret** [comment]
Good idea! If SW5e is a copy paste of dnd5e, it shouldn't be too hard.

**Comment by gravagehulk** [comment]
Thank you. That would be awesome.

**Comment by gravagehulk** [comment]
Just wanted to check in and see if this is still in the works. The SW5e system is a modification of 5e and has a lot of attacks that could take advantage of these effects (i.e. Flame thrower cone, etc). It would be really cool to see the animations in action in that system, as I'm running it starting this week.

Thanks!

**Comment by Feu-Secret** [comment]
Yes. It's still in the work. It's just that I'm busy with other things at the moment. I could get back on TMFX in a couple week.
